index.html

```diff
@@ -88,7 +88,7 @@
 <div style = "opacity:1;animation-name:hello;animation-duration:1.2s;">
        <div id = "info1" style="display:inline;font-size:12.5px;padding-left:3px;color:#f2e7fe;opacity:0.65;"> 
         <b>
-             #DoWhatYouCan't | © <img style = "display:none;" src = "92055249_2886765738070209_5775506505059008512_n.jpg" alt = "Noble Mittal" width="25px" height="19px">Noble Mittal 2020-21 | BN03
+             #DoWhatYouCan't | © <img style = "display:none;" src = "92055249_2886765738070209_5775506505059008512_n.jpg" alt = "Noble Mittal" width="25px" height="19px">Noble Mittal 2020-22 | BN03
```

Comment:
Can we remove `<img>` tag from here as the display has been set to `none`?

Reply:
Done!